src/main/scala/microsites/layouts/FeaturesLayout.scala

```diff
@@ -36,23 +37,23 @@ class FeaturesLayout(config: MicrositeSettings) extends Layout(config) {
   }
 
   def homeHeaderFeatures: TypedTag[String] =
-    div(
-      header(
-        id := "masthead",
-        cls := "features-masthead",
+    header(
+      id := "masthead",
+      cls := "features-masthead",
+      div(
+        cls := "container feature-header",
         div(
-          cls := "container feature-header",
-          div(
-            cls := "features-header-description",
-            h1(cls := "masthead-title", config.identity.name),
-            p(cls := "masthead-description", config.identity.description),
-            a(
-              href := config.gitSiteUrl,
-              cls := "masthead-button",
-              s"View on ${config.gitSettings.gitHostingService.name}")
-          ),
-          div(cls := "features-image"),
+          cls := "features-header-description",
+          h1(cls := "masthead-title", config.identity.name),
+          p(cls := "masthead-description", config.identity.description),
+          a(
+            href := config.gitSiteUrl,
+            target := "_blank",
+            rel := "noopener noreferrer",
+            cls := "masthead-button",
+            s"View on ${config.gitSettings.gitHostingService.name}")
         ),
+        div(cls := "features-image"),
       ),
       "{% if page.position != null %}",
```

Comment:
We need to move the "menu-container" out of the header element the same way we are doing in the `lightHomeHeader` function.

Reply:
Nice catch!

Reply:
Addressed in  2cf6343